src/services/index.js

```diff
@@ -1,3 +1,6 @@
 export { getVideosService } from "./videosServices/getVideos.service";
 export { loginService } from "./authServices/login.service";
 export { signUpService } from "./authServices/signUp.service";
+export { getLikesService } from "./likesServices/getLikes.service";
+export { addToLikesService } from "./likesServices/addToLikes.service";
+export { removeFromLikesService } from "./likesServices/removeFromLikes.service";
```

Comment:
You could create an `index.js` file for services and export all of these from that. It would help in importing all together as named imports rather than importing each one seperately.

Reply:
You are commenting in the indes.js file itself, I have already made that.